### PR TITLE
Feature/icon package

### PR DESCRIPTION
**Pull request by vicentelyrio**
# Description

Creating **Icons** package https://natura.atlassian.net/browse/DSY-315

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# Checklist:

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] I have made corresponding changes to the documentation
- [X] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [X] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: packages/icons/node_modules/.bin/svgo [hunk 1]
```diff
@@ -0,0 +1 @@
+../../../../node_modules/svgo/bin/svgo
```
Comment (robertLichtnow) [inline packages/icons/node_modules/.bin/svgo]: Is there a specific reason for this to be versioned?

File: packages/docs/dist/index.html [hunk 2]
```diff
@@ -1,18 +1,45 @@
-<!DOCTYPE html>
```
Comment (robertLichtnow) [inline packages/docs/dist/index.html]: Rollback this file

File: packages/icons/dist/natds-icons.css [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+
+.natds-icons-9x {
+  font-size: 9em;
+}
+
+.natds-icons-10x {
+  font-size: 10em;
+}
+
+.natds-icons-fw {
```
Comment (robertLichtnow) [inline packages/icons/dist/natds-icons.css]: What does `fw` stand for?

File: packages/icons/scripts/svgCleaner.js [hunk 4]
```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env node
+const fs = require('fs');
+const path = require('path');
+const SVGO = require('svgo');
+
+const dirname = "./src/assets/";
+
+const svgo = new SVGO({
+  plugins: [
```
Comment (robertLichtnow) [inline packages/icons/scripts/svgCleaner.js]: What a nightmare. Good job!

File: packages/icons/dist/natds-icons.css (hunk 3 above)
Comment (vicentelyrio) [reply packages/icons/dist/natds-icons.css]: full-width, these classes are generated by the webfont template, can be customized later

**Pull request merged**